docs.sh

```diff
@@ -4,4 +4,4 @@ set -ex
 cargo doc
 echo "<meta http-equiv=refresh content=0;url=lal/index.html>" > target/doc/index.html
```

Comment:
You might also need https://github.com/lalbuild/lal-build.xyz/blob/master/index.php and https://github.com/lalbuild/lal-build.xyz/blob/master/.buildpacks to satisfy the buildpack.